lib/core/proxy.c

```diff
@@ -336,7 +336,7 @@ static void do_send(struct rp_generator_t *self)
     size_t veccnt;
     h2o_send_state_t ststate;
 
-    assert(self->sending.bytes_inflight == 0);
+    assert(!self->sending.inflight);
```

Comment:
How about removing this assertion now that the check is done in `h2o_doublebuffer_prepare`?

Reply:
fixed at https://github.com/h2o/h2o/pull/1508/commits/bb5b09ea529186ac5e6a9ebb439460b3cb2fad38